src/Login/containers/LoginContainer.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Login from '../components/LoginComponent';
+import { loginUtil } from '../redux/validations';
+import loginAction from '../redux/actions/LoginAction';
+
+export class LoginContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      errors: {
+        invalid_email:false,
+      }
+    };
+  }
+  onChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+
+    const target = event.target;
+    const value = target.type === 'checkbox' ? target.checked : target.value;
```

Comment:
What's the point of this check? I think all you need is the value.

Reply:
l have worked on this as well  thank you.